`dfextall` is broken

```console
$ go build -tags dfextall ./frontend/dockerfile/cmd/dockerfile-frontend
# github.com/moby/buildkit/frontend/dockerfile/instructions
frontend/dockerfile/instructions/commands_secrets.go:5:6: isSecretMountsSupported redeclared in this block
        previous declaration at frontend/dockerfile/instructions/commands_nosecrets.go:5:32
```

I'm not sure how we can fix this.

Maybe we should just eliminate all build tags and use run-time CLI/ENV flags instead?

--- a/frontend/dockerfile/instructions/commands_nosecrets.go
+++ b/frontend/dockerfile/instructions/commands_nosecrets.go
@@ -1,4 +1,4 @@
-// +build !dfsecrets
+// +build !dfsecrets,!dfextall
 
 package instructions
 

--- a/hack/test
+++ b/hack/test
@@ -13,4 +13,7 @@ docker run --rm -v /tmp --privileged $iid go test ${TESTFLAGS:--v} ${TESTPKGS:-.
 
 docker run --rm $iid go build ./frontend/gateway/client
 docker run --rm $iid go build ./frontend/dockerfile/cmd/dockerfile-frontend
+
 docker run --rm $iid go build -tags dfrunmount ./frontend/dockerfile/cmd/dockerfile-frontend
+docker run --rm $iid go build -tags "dfrunmount dfsecrets" ./frontend/dockerfile/cmd/dockerfile-frontend
+docker run --rm $iid go build -tags dfextall ./frontend/dockerfile/cmd/dockerfile-frontend

